views/deductions/trillium-energy-cost.pug

```diff
@@ -11,6 +11,8 @@ block content
   form.cra-form(method='post')
     +radiosYesNo('trilliumEnergyCostClaim', 'In 2018, when you lived on an Ontario reserve, did you pay home energy costs, for example, to heat or cool your home, or use appliances like a stove or fridge?', trilliumEnergyCostClaim)
 
+    include ../_includes/principalResidence
```

Comment:
Was this left out of a previous PR?

Reply:
Ah, yes. I forgot to note it in my comment as I noticed it the other day and it slipped my mind today. It is supposed to be on this page per the content doc, but got missed in the PR for that page.